src/sync-endpoint-cloud-setup.rst

```diff
@@ -434,74 +449,94 @@ Setting up an AWS account
 Setting up a virtual machine
 """""""""""""""""""""""""""""
 
-1. First, click on :guilabel:`EC2` link under the **COMPUTE** section. Then, go ahead and launch a new instance.
+1. In your AWS console, search for **EC2** in the search bar and choose :guilabel:`EC2` under the **Services** section. Then, click :guilabel:`Launch instance` to start creating your instance.
 
   .. image:: /img/setup-aws/aws1.png
    :width: 600
 
   .. image:: /img/setup-aws/aws2.png
    :width: 600
 
-2. You must start by choosing an Amazon Machine Image (AMI). Scroll through the options and select *Ubuntu Server 18.04 LTS (HVM), SSD Volume Type* which should be the fifth option from the top.
+2. Enter a name for the instance in the :guilabel:`Name` field. Then select **Ubuntu Server 22.04 LTS (HVM), SSD Volume Type** as the Amazon Machine Image (AMI) and **t2.micro** as the instance type.
 
   .. image:: /img/setup-aws/aws3.png
    :width: 600
 
-3. Skip the “Choose an Instance Type” step. Instead, click on the :guilabel:`3: Configure Instance` tab at the top and then attach the :download:`cloud_init_AWS.yml</files/cloud_init_AWS.yml>` file we provided within the **User data** section under “Advanced Details.”
+  .. image:: /img/setup-aws/aws4.png
+  :width: 600
 
-|
+  .. image:: /img/setup-aws/aws5.png
+  :width: 600
 
-4. Click on the :guilabel:`6. Configure Security Group` tab in order to modify the firewall rules and control the traffic for the instance. Create a new security group and modify the rules to match the rules specified below, then click :guilabel:`Review and Launch`.
+3. Now, generate a key pair for connecting to your instance. Click on :guilabel:`Create new key pair`, enter a name, choose type and format, then click :guilabel:`Create key pair`. Your key pair will be automatically downloaded to your local computer, ensure you store it in a secure location.
 
-  .. image:: /img/setup-aws/aws4.png
-   :width: 600
+  .. image:: /img/setup-aws/aws6.png
+  :width: 600
 
-5. Review the Instance Launch and then click :guilabel:`Launch`. Now, create a new key pair to access your instance via SSH and make sure to download it to a secure location. Finally, click :guilabel:`Launch Instances`!
+  .. image:: /img/setup-aws/aws7.png
+  :width: 600
 
-  .. image:: /img/setup-aws/aws5.png
-   :width: 600
+4. In the **Network Settings** section, click on :guilabel:`Edit`. Select :guilabel:`Create security group` and provide a name and description for the security group. Then add inbound rules for the security group as shown in the images below.
```

Comment:
Is it possible to explain these in text as well or is it too long/difficult?

Reply:
Done